client/components/login/login.js

```diff
@@ -174,31 +179,66 @@ export default class Login extends React.Component {
                   </div>
                 )}
 
-                <div className="row email">
-                  <label htmlFor="email">
-                    {getText(input_fields.email.label, language)}
-                  </label>
-                  {errors.email && (
-                    <div className="error">
-                      <span className="icon">!</span>
-                      <span className="text">
-                        {errors.email}
-                      </span>
-                    </div>
-                  )}
-                  <input
-                    className={`input ${errors.email ? "error" : ""}`}
-                    type={input_fields.email.type}
```

Comment:
please ensure the configuration mirrors and reuses the same configuration used for the registration component! Do as it's done in the `mobile-phone-change` component, if you inspect that component (check `index.js`) you'll see the configuration is taken from the registration component:

https://github.com/openwisp/openwisp-wifi-login-pages/blob/f4fee4b1ab52156d3026296120ba92c1d1c2f1d5/client/components/mobile-phone-change/index.js#L9-L11

Ensure also the HTML is exactly the same as in the change phone number and registration component.